### PR TITLE
Better organized 3.6 changelog 

### PR DESCRIPTION
**Pull request by dereksmart**
Featured images first, to match the release post.  

Up for discussion.

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -102,19 +102,23 @@ Learn more (and get detailed instructions) in our [contribute guide](http://jetp
...
 
-* Enhancement: CLI: Add a number of Jetpack CLI improvements: see http://jetpack.me/support/jetpack-cli
+Feature Enhancements:
+* CLI: Add a number of Jetpack CLI improvements: see http://jetpack.me/support/jetpack-cli
+* New Jetpack admin page for connection management
+* New Social Media Icons widget
+* FB Like Box: A visual refresh of the Facebook likebox widget
+* Protect: When your IP is blocked, use a math captcha as a fallback instead of a complete block
+
+Full list of changes:
```
Comment (kraftbj) [inline readme.txt]: /s/full list of/Additional since we're taking the items above out of the list below.

File: readme.txt [hunk 2]
```diff
@@ -102,19 +102,23 @@ Learn more (and get detailed instructions) in our [contribute guide](http://jetp
 
 = 3.6 =
 Release Date: July 6th, 2015
+Release Post: http://jetpack.me/2015/07/06/jetpack-3-6-updated-social-widgets-jetpack-cli-and-more/
```
Comment (georgestephanis) [inline readme.txt]: Any chance of backfilling this into prior releases as well?

File: readme.txt (hunk 1 above)
Comment (dereksmart) [inline readme.txt]: makes sense!

File: readme.txt (hunk 2 above)
Comment (dereksmart) [inline readme.txt]: might as well while we're at it :)

Comment (dereksmart) [inline readme.txt]: @georgestephanis looks like prior major releases already have the post linked.  Did you mean changing them to highlight featured changes first?

**Pull request merged**